pkg/api/validation/validation.go

```diff
@@ -1921,6 +1921,10 @@ func ValidateSecurityContext(sc *api.SecurityContext) errs.ValidationErrorList {
 			allErrs = append(allErrs, errs.NewFieldInvalid("runAsUser", *sc.RunAsUser, "runAsUser cannot be negative"))
 		}
 	}
+
+	if sc.SELinuxOptions != nil && !capabilities.Get().EnableSELinuxIntegration {
+		allErrs = append(allErrs, errs.NewFieldForbidden("selinuxOptions", "Cannot specify selinuxOptions if SELinux integration is not enabled"))
```

Comment:
Why is this an error condition in both validation and the kubelet.  If I submit a pod with seLinuxOptions and the system isn't enforcing does it matter?  Shouldn't my pod run anyway since the system is less restrictive?